src/reducers/schemas/selectors.js

```diff
@@ -1,42 +1,140 @@
 import { createSelector } from 'reselect'
```

Comment:
I personally think it makes sense to keep reducers and selectors close (as in same file), https://twitter.com/dan_abramov/status/730933179511640064, based on Dan's recommendations (not my own idea). In general I think what's going on with the selectors is a bit abstract, so whatever you can do to make this friendlier to newcomers to the codebase would be nice (as in explanatory comments, etc.).

Reply:
I think we should address this in a different PR

Reply:
Yeah it seems like this wasn't introduced in this PR, so that makes sense 👍